<launch>
    <arg name="suffix" default=""/>
    <arg name="max_position_error" default="0.002"/>
    <arg name="timeout" default="5.0"/>
    <arg name="closing_time" default="2.0"/>
    <arg name="rate" default="5"/>
    <arg name="tolerance" default="0.002"/>

    <node pkg="pal_parallel_gripper_wrapper" name="gripper$(arg suffix)_grasping" type="gripper_grasping.py" output="screen">
        <rosparam command="load" file="$(find pal_parallel_gripper_wrapper)/config/parallel_gripper$(arg suffix).yaml" />
        <!-- Dynamic reconfigure parameters -->
        <param name="grasper/max_position_error" type="double" value="$(arg max_position_error)"/>
        <param name="grasper/timeout" type="double" value="$(arg timeout)"/>
        <param name="grasper/closing_time" type="double" value="$(arg closing_time)"/>
        <param name="grasper/rate" type="int" value="$(arg rate)"/>
        <param name="grasper/tolerance" type="double" value="$(arg tolerance)"/>
    </node>
</launch>
